### PR TITLE
On landingpage add buttons to examples

### PR DESCRIPTION
**Pull request by prakhargupta1**
Screenshot 2023-06-28 at 10 02 16 AM

https://deploy-preview-2196--mui-toolpad-docs.netlify.app/toolpad/#:~:text=Upvote-,Use%20cases,-Toolpad%20is%20ideal

### CODE REVIEW

**Comment by Janpot** [comment]
The button doesn't work for me

File: docs/src/components/landing/UseCases.js [hunk 1]
```diff
@@ -36,17 +38,39 @@ export default function CardGrid(props) {
...
                   <Typography variant="body" color="text.secondary">
                     {description}
                   </Typography>
+                  {action ? (
+                    <Button
+                      sx={{ py: 1, width: 'fit-content' }}
+                      variant="contained"
+                      endIcon={<KeyboardArrowRightRounded />}
+                      target="_blank"
+                      rel="noopener"
```
Comment (Janpot) [inline docs/src/components/landing/UseCases.js:68]:
```suggestion
```

**Comment by prakhargupta1** [comment]
> All three links point to the same page

Yes, Docs page for #2096 and #2151 is to be created after these are merged, and those will be the links for the **View example** button in this PR

**Review by Janpot — APPROVED** [review]

**Comment by prakhargupta1** [comment]
@Janpot There is a Firefox test failing after merging the master to this branch. What should be done in this case?

**Pull request merged**